src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -35,7 +36,8 @@ internal partial class CurlHandler : HttpMessageHandler
         private const string UriSchemeHttps = "https";
         private const string EncodingNameGzip = "gzip";
         private const string EncodingNameDeflate = "deflate";
-        private readonly static string[] AuthenticationSchemes = { "Negotiate", "Digest", "Basic" }; // the order in which libcurl goes over authentication schemes
+        private readonly static string[] AuthenticationSchemes = { "Negotiate", "Digest","NTLM", "Basic" }; // the order in which libcurl goes over authentication schemes
```

Comment:
Remove NTLM as scheme.  The list should only contain 'Negotiate', 'Digest', 'Basic'